Controls/Input/Input.UWP/NumericBox/RadNumericBox.cs

```diff
@@ -766,7 +767,7 @@ private static void OnValueStringChanged(DependencyObject d, DependencyPropertyC
 
         private static bool IsNumericKey(VirtualKey key)
         {
-            if (RadNumericBox.IsAzertyKeyboard && key == VirtualKey.Number6)
+            if (RadNumericBox.IsAzertyKeyboard && key == VirtualKey.Number6 && AnalyticsInfo.VersionInfo.DeviceFamily != "Windows.Mobile")
```

Comment:
As we are using the string literal "Windows.Mobile" at two places for an easy maintenance I could suggest you to declare it as a constant and use it instead.

For example right after the "DotKey" const field you could add one called:
`private const string DeviceFamilyMobileName = "Windows.Mobile";`
and use it instead of the string literal:
` if (RadNumericBox.IsAzertyKeyboard && key == VirtualKey.Number6 && AnalyticsInfo.VersionInfo.DeviceFamily != DeviceFamilyMobileName)`

Thus we could maintain the class easier if we need to change it in the future as it is used at several places.